packages/tools/devtools/devtools-core/src/data-visualization/DefaultEditors.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This module contains default {@link EditSharedObject}
+ * implementations for our DDSs.
+ */
+
+import { SharedCell } from "@fluidframework/cell";
+import { SharedCounter } from "@fluidframework/counter";
+import { SharedString } from "@fluidframework/sequence";
+
+import { ISharedObject } from "@fluidframework/shared-object-base";
+import { Serializable } from "@fluidframework/datastore-definitions";
+import { EditSharedObject } from "./DataEditing";
+
+/**
+ * Default {@link EditSharedObject} for {@link SharedCell}.
+ */
+export const editSharedCell: EditSharedObject = async (
+	sharedObject: ISharedObject,
+	newData: Serializable,
+): Promise<void> => {
+	const sharedCell = sharedObject as SharedCell<unknown>;
+	const data = sharedCell.get();
+	console.log(data);
+};
+
+/**
+ * Default {@link VisualizeSharedObject} for {@link SharedCounter}.
+ */
+export const editSharedCounter: EditSharedObject = async (
+	sharedObject: ISharedObject,
+	newData: number,
+): Promise<void> => {
+	const sharedCounter = sharedObject as SharedCounter;
+	sharedCounter.increment(Math.floor(newData) - sharedCounter.value);
+	console.log(sharedCounter);
+};
+
+/**
+ * Default {@link EditSharedObject} for {@link SharedString}.
+ */
+export const editSharedString: EditSharedObject = async (
+	sharedObject: ISharedObject,
+	newData: string,
+): Promise<void> => {
+	const sharedString = sharedObject as SharedString;
+	if (newData === "") {
+		sharedString.removeText(0, sharedString.getLength());
+	} else {
+		sharedString.replaceText(0, sharedString.getLength(), newData);
+	}
+	return;
+};
+
+/**
+ * {@link EditSharedObject} for unrecognized {@link ISharedObject}s.
+ */
+export const editUnknownSharedObject: EditSharedObject = async (
```

Comment:
This shouldn't be needed. When we generate the visualization tree for unknown objects, we will not annotate them as being editable, so we should never receive an edit for something unknown. And if we do, the logic in `DataVisualization` can just drop it.

Reply:
How would I edit [image below] without having some kind of default? I need to throw some error but can't quite get that bit of code to act the way I want
![image](https://github.com/microsoft/FluidFramework/assets/134635443/250a8321-389c-412f-a856-e876400f251d)

Reply:
For editing, it would probably look something like:
```typescript
if (this.editors[sharedObject.attributes.type] === undefined) {
    // error
} else {
    // call edit function}
```

Reply:
Check new commit